tests/test_hdate_api.py

```diff
@@ -220,3 +220,20 @@ def test_zmanim_localized_datetime(self) -> None:
             date=datetime(2019, 4, 21, 20, 30, tzinfo=_timezone), location=coord
         )
         assert not zman.issur_melacha_in_effect
+
+    def test_candle_lighting_localized_datetime(
+        self, capsys: CaptureFixture[str]
+    ) -> None:
+        """Test for issur melacha if datetime is localized."""
+        coord = Location(
+            name="New York",
+            latitude=40.7128,
+            longitude=-74.0060,
+            timezone="America/New_York",
+            diaspora=True,
+        )
+        _timezone = cast(tzinfo, coord.timezone)
+        zman = Zmanim(date=datetime(2024, 10, 18, tzinfo=_timezone), location=coord)
+        print(zman.candle_lighting)
```

Comment:
No need to use capsys, print and compare.
Just compare zman.candle_lighthing directly to the correct datetime